datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/mirrormaker/TestKafkaMirrorMakerConnectorTask.java

```diff
@@ -366,6 +371,55 @@ boolean isPostShutdownHookExceptionCaught() {
     }
   }
 
+  @Test
+  public void testPartitionManagedLockAcquireFailMetric() throws InterruptedException {
+    String datastreamName = "pizzaStream";
+    Datastream datastream = KafkaMirrorMakerConnectorTestUtils.createDatastream(datastreamName, _broker, "\\w+Pizza");
+    DatastreamTaskImpl task = spy(new DatastreamTaskImpl(Collections.singletonList(datastream)));
+    doThrow(DatastreamRuntimeException.class).when(task).acquire(any(Duration.class));
+    MockDatastreamEventProducer datastreamProducer = new MockDatastreamEventProducer();
+    task.setEventProducer(datastreamProducer);
+
+    KafkaBasedConnectorConfig connectorConfig = new KafkaBasedConnectorConfigBuilder()
+        .setConsumerFactory(new LiKafkaConsumerFactory())
+        .setCommitIntervalMillis(10000)
+        .setEnablePartitionManaged(true)
+        .build();
+
+    ZkAdapter zkAdapter = new ZkAdapter(_kafkaCluster.getZkConnection(), "testCluster", null,
+        ZkClient.DEFAULT_SESSION_TIMEOUT, ZkClient.DEFAULT_CONNECTION_TIMEOUT, null);
+    task.setZkAdapter(zkAdapter);
+    zkAdapter.connect();
+
+    String connectorName = "KafkaMirrorMaker";
+    KafkaMirrorMakerConnectorTaskTest connectorTask = new KafkaMirrorMakerConnectorTaskTest(connectorConfig, task, connectorName,
+        false, new KafkaMirrorMakerGroupIdConstructor(false, "testCluster"));
+    // We don't want to wait for the task to start, since it will throw before the start countdown latch can be downed.
+    AtomicReference<Throwable> throwable = new AtomicReference<>();
+    Thread connectorThread =
+        KafkaMirrorMakerConnectorTestUtils.runKafkaMirrorMakerConnectorTask(connectorTask, (t1, e) -> throwable.set(e),
+            false);
+    connectorThread.join();
+
+    Assert.assertEquals(DatastreamRuntimeException.class, throwable.get().getClass());
+
+    // verify that the metric to indicate task lock acquire errors is incremented
+    Meter metric = DynamicMetricsManager.getInstance()
+        .getMetric(connectorName + "." + KafkaMirrorMakerConnectorTask.class.getSimpleName() + "." + datastreamName
+            + "." + "taskLockAcquireErrorRate");
+    Assert.assertNotNull(metric);
+    Assert.assertEquals(metric.getCount(), 1);
+
+    // Verify that metrics created through DynamicMetricsManager match those returned by getMetricInfos() given the
```

Comment:
nit: Feel free to ignore it. Can you please simplify this?

Reply:
Simplify the comment or code?

Reply:
just the comment.

Reply:
I'll skip this as I can't think of a way to simplify it.